cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -146,7 +146,7 @@ public AbstractConnectOperator(Vertx vertx, PlatformFeaturesAvailability pfa, St
         this.port = port;
 
         // Setup metrics for connectors
-        Tags metricTags = Tags.of(Tag.of("kind", KafkaConnector.RESOURCE_KIND));
+        Tags metricTags = Tags.of(Tag.of("kind", KafkaConnector.RESOURCE_KIND), Tag.of("selector", ""));
```

Comment:
So Kafka connect does have this selector always empty?

Reply:
this is KafkaConnector not KafkaConnect

Reply:
@sknot-rh Hello, we are keeping it as empty always because of the prometheus meter property. If we dont put the selector here then, it will lead to something like this  --->`Prometheus requires that all meters with the same name have the same set of tag keys. There is already an existing meter named 'strimzi_reconciliations_total' containing tag keys [kind, selector]. The meter you are attempting to register has keys [kind].`

Reply:
If your question is why we are using an empty selector label.

Reply:
Ah, thanks @ppatierno